tutorials/plot_artifacts_correction_ica.py

```diff
@@ -76,20 +76,19 @@
 # Component properties
 # --------------------
 #
-# Let's take a closer look at three potential candidates for artifact-related
-# components: IC 12, IC 15 and IC 21
+# Let's take a closer at properties of first three independent components
 
-# first, component 12:
-ica.plot_properties(raw, picks=12, dB=True)
+# first, component 0:
+ica.plot_properties(raw, picks=0)
 
 ###############################################################################
-# it looks like a blink component, but because the data were filtered
-# the spectrum plot is not very informative, let's change that:
-ica.plot_properties(raw, picks=12, dB=True, psd_args={'fmax': 35.})
+# we can see that the data were filtered so the spectrum plot is not
+# very informative, let's change that:
+ica.plot_properties(raw, picks=0, psd_args={'fmax': 35.})
 
 ###############################################################################
-# now let's inspect properties of components 15 and 21 (cardiac activity):
-ica.plot_properties(raw, picks=[15, 21], psd_args={'fmax': 35.})
+# we can also take a look at multiple different components at once:
+ica.plot_properties(raw, picks=[1, 2], psd_args={'fmax': 35.})
```

Comment:
how did we call this parameter elsewhere in the ICA code btw.? I think not picks. We should check and open an issue if needed to fix it.